### ISSUE TITLE
getCompletion example is not working

### ISSUE THREAD

**Issue by agrakhov** [yargs/yargs#1434]
I'm trying to play with `.getCompletion` but not sure if its working correctly. If I'm following the [example in docs](https://github.com/yargs/yargs/blob/master/docs/api.md#getcompletionargs-done) and have this in `test.js`:
```
#! /usr/bin/env node
require('yargs')
	.option('foobar')
	.option('foobaz')
	.completion()
	.getCompletion(['./test.js', '--foo'], function(completions) {
		console.log(completions)
	})
// I've tried to add .argv at the end - not helping
```
when I try to run `./test.js --foo` following error is reported:
```
.../test-cli/node_modules/yargs/lib/completion.js:76
        const keyAndAliases = [key].concat(aliases[key] || [])
                                                  ^

TypeError: Cannot read property 'help' of undefined
    at Object.keys.forEach (.../test-cli/node_modules/yargs/lib/completion.js:76:51)
    at Array.forEach (<anonymous>)
    at Object.getCompletion (.../test-cli/node_modules/yargs/lib/completion.js:74:43)
    at Object.Yargs.self.getCompletion (.../test-cli/node_modules/yargs/yargs.js:933:16)
    at Object.<anonymous> (.../test-cli/test.js:6:3)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

I hope its just incorrect example in docs )

**Comment by JakeRowsell89** [comment]
I'm able to reproduce. Out of curiosity, are you on zshell? There seems to be some zshell specific code

**Comment by agrakhov** [comment]
@JakeRowsell89 yep I'm using ohmyzsh.

**Comment by mleguen** [comment]
@agrakhov @JakeRowsell89 

This should be fixed now in master.

**Comment by mleguen** [comment]
Fixed in 15.1.0
